e3nn/o3/s2grid.py

```diff
@@ -265,6 +276,8 @@ class ToS2Grid(torch.nn.Module):
     lmax : int
     res : int, tuple of int
     normalization : {'norm', 'component', 'integral'}
+    dtype : torch.dtype or None, optional
+    device : torch.device or None, optional
```

Comment:
Should these parameters have a comment like in `s2_grid` above?

Reply:
I agree, but the other argumetns do not have such long comments so I've followed the style.

Reply:
Removed in jnooree/e3nn@bf83b042eadf08d5a1c8d452eb905ffd6b909e99